prog/vm/nexus.rb

```diff
@@ -294,8 +294,17 @@ def trigger_refresh_mesh
 
   def run
     host.sshable.cmd("sudo systemctl start #{q_vm}")
-    vm.update(display_state: "running")
-    hop :wait
+    hop :wait_sshable
+  end
+
+  def wait_sshable
+    addr = vm.ephemeral_net4 || vm.ephemeral_net6.nth(2)
+    out = `ssh -o BatchMode=yes -o ConnectTimeout=1 -o PreferredAuthentications=none user@#{addr} 2>&1`
```

Comment:
I just wonder can we do it with `net-ssh` package instead of running command on host machine? I think we don't have any example that run commands on host machine. If we can, it might be good practice.

Reply:
I think I prefer the simplicity of the current implementation. We are just running an ssh command that is doomed to fail. We only care about a single output. I also tried the same with ping to make it even simpler however, it doesn't validate the ssh access.

Reply:
I'm fine with the current approach. If Enes does not feel strongly about it, we can merge it.

Also did you mean control plane? This command doesn't run on host machine, right?

Reply:
I'm not that strong.

Yeah I mean machine that we deployed CP as host machine. Control plane's host